flink-libraries/flink-gelly/src/main/java/org/apache/flink/graph/library/linkanalysis/HITS.java

```diff
@@ -168,7 +168,6 @@ protected void mergeConfiguration(GraphAlgorithmWrappingBase other) {
 				.setParallelism(parallelism)
 				.name("Square")
 			.reduce(new Sum())
-			.setCombineHint(CombineHint.HASH)
```

Comment:
would have expected a separate commit but ok - I can't say much about the change though since I don't know flink's graph API too much...